components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -102,6 +103,21 @@ protected void initiateAuthenticationRequest(HttpServletRequest request,
         String password = (String) context.getProperty(PASSWORD_PROPERTY);
         context.getProperties().remove(PASSWORD_PROPERTY);
 
+        Map<String, String> runtimeParams = context.getAuthenticatorParams(FrameworkConstants.JSAttributes.JS_COMMON_OPTIONS);
```

Comment:
Shall we update getRuntimeParams() method to include the common params also?

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/1694